tests/modules/nf-core/gatk4/splitcram/test.yml

```diff
@@ -0,0 +1,9 @@
+- name: gatk4 splitcram test_gatk4_splitcram
+  command: nextflow run ./tests/modules/nf-core/gatk4/splitcram -entry test_gatk4_splitcram -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/gatk4/splitcram/nextflow.config
+  tags:
+    - gatk4
+    - gatk4/splitcram
+  files:
+    - path: output/gatk4/test.0000.cram
```

Comment:
Probably the file is too small, but would it be possible to actually split the file in two? was trying to find a suitable argument but did not...

Reply:
Fixed it with `--shard-records 2` ;)